internal/symbolizer/symbolizer.go

```diff
@@ -75,7 +80,7 @@ func (s *Symbolizer) Symbolize(mode string, sources plugin.MappingSources, p *pr
 	var err error
 	if local {
 		// Symbolize locally using binutils.
-		if err = localSymbolize(mode, p, s.Obj, s.UI); err != nil {
+		if err = localSymbolize(p, fast, force, s.Obj, s.UI); err != nil {
```

Comment:
Curious: looks like there is no way to force remote symbolization so it's always either forced or always skips already symbolized mappings?

Reply:
Not sure I understand the question.

-symbolize=force will force both remote and local symbolization.
-symbolize=force:remote will force remote symbolization.

Reply:
Oh I see what you mean. Yes, symbolz will always skip mappings that already have function names. 
I'll propagate the option and check there too, just for consistency.